src/libs/ajax/AzureStorage.ts

```diff
@@ -111,6 +111,49 @@ export const AzureStorage = (signal?: AbortSignal) => ({
     }), notebooks)
   },
 
+  blobMetadata: (azureStorageUrl: string) => {
+    const getObjectMetadata = async () => {
+      const workspaceId = azureStorageUrl.split('/')[3].replace('sc-', '')
```

Comment:
thought of doing this with regex for matching on the guid/sc-guid but in theory someone could name a folder or file a similar way and then we would get the wrong workspace id. would take any other ideas/suggestions gladly!